app/common/src/cljc/teet/project/project_model.cljc

```diff
@@ -92,6 +92,15 @@
 
 (defmulti get-column (fn [_project column] column))
 
+(defmulti get-column-compare (fn [column] column))
+
+(defmethod get-column-compare :default [_] nil)
+
+(defmethod get-column-compare :thk.project/activity-status [_]
+  (fn [x y] (compare
```

Comment:
I don't understand what is the need for this new multi method.

And why are we comparing two keyword idents together? Is there a sort order for the keywords or should they be compared based on the translated labels?

Reply:
The need for the multimethod is to provide compare for the particular column (or columns in general)
How it should be sorted is not specified in the issue. Will ask ...